Cargo.toml

```diff
@@ -13,9 +13,22 @@ readme = "README.md"
 [dependencies]
 data-encoding = "2.1"
 reqwest = "0.9"
+futures = "^0.1"
 serde = "1.0"
 serde_derive = "1.0"
 serde_json = "1.0"
 url = "1.7"
 failure = "0.1.2"
 failure_derive = "0.1.2"
+
+[dev-dependencies]
+tokio = "^0.1"
+
+[features]
+# TODO: Deprecate `async` after we can use Generic Associated Types
```

Comment:
Can you explain what you mean by this comment?

Reply:
I admit that I just copy paste the line from #47 and I forgot to remove this comment.